src/components/app/ProfileViewer.css

```diff
@@ -29,12 +29,13 @@
   margin: 3px;
   padding: 0;
   border: 1px solid var(--green-60);
+  background: url(../../../res/back-arrow.svg) center center no-repeat;
```

Comment:
nit: this `background` is disconnected from below `background-color` (I noticed because I initially thought there were no `background-color` and I wanted to outline it. So this definitly confused me).
Please either move the `background-color` value directly in this rule, or put the `background` in the same "group" below next to `background-color`. Either of these choices would work for me.